Apps/phdnsdb/readme.html

```diff
@@ -0,0 +1,264 @@
+<!-- File: readme.html
+  Copyright (c) 2016-2021 Splunk Inc.
+
+Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+-->
+<html>
+  <body>
+    <p><h2>Description Action Parameters</h2>
+        <ol>
+          <h3 class="lowercase"><li>Test Connectivity (Action Workflow)</li></h3>
+            <ul>
+              <li>This action will test the connectivity with the dnsdb server using the provided API key value through the rate_limit API.[https://api.dnsdb.info/dnsdb/v2/rate_limit]</li>
+              <li>The action  also validates the required asset parameter values and based on the value API response will be displayed.</li>
+              <li>For successful test connectivity, the query quota details would be shown as “Test succeeded. Query quota is 1000 with 987 queries remaining. Resets 1618531200”</li>
+            </ul>
+            <br>
+
+          <h3 class="lowercase"><li>RDATA Name Lookup</li></h3>
+            <ul>
+              <li><b><u>Action Parameter</u>​ - name</b>
+                <ul>
+                  <li>The value is a DNS domain name in presentation format("www.example.com"), or a left-hand (“.example.com”) or right-hand (“www.example.”) wildcard domain name. Note that left-hand wildcard queries are somewhat more expensive than right-hand wildcard queries.</li>
+                  <li>Example: www.farsightsecurity.io</li>
+                </ul>
+              </li>
+              <li><b><u>Action Parameter</u>​ - limit</b>
+                <ul>
+                  <li>Limit for the number of results returned via these lookup methods. The default value of limit is ‘200’</li>
+                  <li>For limit=0 , maximum number of results will retrieved.[ here max = results_max of rate_limit].</li>
+                </ul>
+              </li>
+            </ul>
+          <br>
+          <h3 class="lowercase"><li>RDATA IP Lookup</li></h3>
+            <ul>
+              <li><b><u>Action Parameter</u>​ - ip</b>
+                <ul>
+                  <li>The value is one of an IPv4 or IPv6 single address, with a prefix length, or with an address range. If a prefix is provided, the delimiter between the network address and prefix length is a single comma (“,”) character rather than the usual slash (“/”) character to avoid clashing with the HTTP URI path name separator.</li>
```

Comment:
Please mention this prefix scenario separately in the action note section as we are describing the internal code workflow.

Reply:
Fixed